### PR TITLE
Exlm 881 community exclude user and fixes applied for shimmer

### PR DESCRIPTION
**Pull request by Nehaadobe**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID:

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/curated-cards
- After: https://exlm-881-community-exclude-user--exlm--adobe-experience-league.hlx.live/en/curated-cards

### CODE REVIEW

File: scripts/browse-card/browse-cards-delegate.js [hunk 1]
```diff
@@ -68,7 +68,15 @@ const BrowseCardsDelegate = (() => {
    */
   const handleCoveoService = async () => {
     const facets = [
-      ...(param.contentType ? [{ id: 'el_contenttype', type: 'specific', currentValues: param.contentType }] : []),
+      ...(param.contentType
+        ? [
+            {
+              id: 'el_contenttype',
+              type: param.contentType[0] === 'community' ? 'hierarchical' : 'specific',
```
Comment (noormd08) [inline scripts/browse-card/browse-cards-delegate.js:75]: Keep constant value for community

File: blocks/authorable-card/authorable-card.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default async function decorate(block) {
         .handleArticleDataService(link)
         .then(async (data) => {
           block.querySelectorAll('.shimmer-placeholder').forEach((el) => {
-            el.classList.add('hide-shimmer');
+            el.remove();
```
Comment (noormd08) [inline blocks/authorable-card/authorable-card.js:65]: Do we need a loop for this to remove? Is there any content presented in it?

File: scripts/browse-card/browse-card-placeholder.js [hunk 3]
```diff
@@ -27,21 +23,13 @@ export default class BuildPlaceholder {
...
 
   hide() {
-    this.shimmer.querySelector('.shimmer-placeholder').classList.add('hide-shimmer');
-  }
-
-  setParent(contentDiv) {
-    this.shimmer.appendChild(contentDiv);
+    this.shimmer.remove();
   }
 }
```
Comment (noormd08) [inline scripts/browse-card/browse-card-placeholder.js:35]: Keep the methods as add / remove. Show and hide will get confused when we having other methods.

File: blocks/authorable-card/authorable-card.js (hunk 2 above)
Comment (Nehaadobe) [reply blocks/authorable-card/authorable-card.js]: fixed.

File: scripts/browse-card/browse-cards-delegate.js (hunk 1 above)
Comment (Nehaadobe) [reply scripts/browse-card/browse-cards-delegate.js]: fixed.

File: scripts/browse-card/browse-cards-constants.js [hunk 4]
```diff
@@ -50,3 +50,24 @@ export const ROLE_OPTIONS = Object.freeze({
   LEADER: 'Leader',
   USER: 'User',
 });
+
+export const COMMUNITY_SEARCH_FACET = Object.freeze({
+  children: [
```
[the remark this answers is not in the document]
Comment (Nehaadobe) [reply scripts/browse-card/browse-cards-constants.js]: fixed.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**